sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/BinaryComparisonSimplificationSuite.scala

```diff
@@ -122,4 +127,49 @@ class BinaryComparisonSimplificationSuite extends PlanTest with PredicateHelper
 
     comparePlans(optimized, correctAnswer)
   }
+
+  test("Simplify null and nonnull with filter constraints") {
+    Seq('a === 'a, 'a <= 'a, 'a >= 'a, 'a < 'a, 'a > 'a).foreach { condition =>
```

Comment:
nit: can't we avoid to use symbol? See https://github.com/databricks/scala-style-guide#symbol

Reply:
Thanks for the remind. Is there any simply way to instead of symbol ?

Reply:
At the very least you can do `Symbol(...)`. This isn't a nit because we should support Scala 2.13.

Reply:
Oh I see. The scala-style-guide has the new update. Thansk again.